src/include/miopen/fusion/solvers.hpp

```diff
@@ -277,6 +277,40 @@ struct ConvBinWinogradRxSf2x3g1Fused final : FusionSolverBase
     GetSolution(const FusionContext& context, const FusionDescription& problem) const override;
 };
 
+template <uint32_t Winodata, uint32_t Winofilter>
+struct ConvWinoFuryRxSFused final : FusionSolverBase
+{
+    const std::string& SolverDbId() const override
+    {
+        return GetSolverDbId<ConvWinoFuryRxSFused<Winodata, Winofilter>>();
+    }
+
+    MIOPEN_INTERNALS_EXPORT bool IsApplicable(const FusionContext&,
+                                              const FusionDescription&) const override;
+    bool IsDynamic() const override { return true; }
```

Comment:
Add `MIOPEN_INTERNALS_EXPORT` here. It is required for explicit template instantiation below (line 307).